### DISCUSSION TITLE
155

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#94] [General]
# 潮流周刊第155期 - 东京夜景

封面图来源于在世界第一高塔的东京晴空塔上拍东京夜景，上周在东京玩了几天（哈哈哈，这也是上周鸽的原因…），除去人多之外，都还挺玩的。

https://weekly.tw93.fun/posts/155-%E4%B8%9C%E4%BA%AC%E5%A4%9C%E6%99%AF

**Comment by maxhuangping** [comment]
Hello~大佬~
rss好像出了点问题，盼修复

**Reply by AkianZhou** [reply]
哈喽

**Reply by tw93** [reply]
感谢反馈啊，已经修复，混入了一个特殊字符导致渲染出错了

![](https://gw.alipayobjects.com/zos/k/ye/cEv7tS.png)

**Comment by mggger** [comment]
哈喽，能自荐下做的一个信息收集网站吗？

https://chasetrend.news/

每日更新, 专为开发者设计, 内容专为独立开发者和科技爱好者量身定制, 集成Hacker News、GitHub Trends和Google Trends的最热门内容
